migration/v0.18.x-v0.19.x/upgrade-apps.md

```diff
@@ -2,6 +2,9 @@
 
 ## Steps
 
+1. Run the migration script `remove_deleted_rules_prometheus_alerts.sh` to remove the old Prometheus rules and alerts from both clusters
+> **_WARNING:_** this will "hide" all the alerts until you run the `bin/ck8s apply` and recreate the rules and alerts
+
```

Comment:
Is it not enough to just run `ck8s apply`? Do you need to first remove the old before applying the new?

Reply:
yes, in my test running directly `ck8s apply` failed to remove the old alerts

Reply:
Ok, thanks